packages/engine/src/gltf/GLTFComponent.tsx

```diff
@@ -164,6 +164,12 @@ const loadDependencies = {
       key: 'dependencies',
       eval: (dependencies?: ComponentDependencies) => componentDependenciesLoaded(dependencies)
     }
+  ],
+  ['EE_collider']: [
```

Comment:
For both this jsonID and the other existing once, we should pull them directly from their component definitions (eg `ColliderComponent.jsonID`)